fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/InSubquery.java

```diff
@@ -70,7 +71,7 @@ public Expression getCompareExpr() {
     @Override
     public Expression withChildren(List<Expression> children) {
         Preconditions.checkArgument(children.size() == 2);
-        Preconditions.checkArgument(children.get(0) instanceof Expression);
+        Preconditions.checkArgument(children.get(0) != null);
```

Comment:
I think this uncheck is unneccessary, it's already checked in the constructor of InSubQuery

Reply:
do not check in constructor